c_api/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cpp_demangle_c_api"
+version = "0.1.0"
+authors = ["Miguel Saldivar <saldivarcher@gmail.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+name = "cpp_demangle_c_api"
+path = "lib.rs"
+
+[dependencies]
+cpp_demangle = { path = ".." }
+libc = "0.2"
```

Comment:
We shouldn't need `libc` here, I don't think. All the C types exist in the `std::os::raw` module, eg https://doc.rust-lang.org/nightly/std/os/raw/type.c_char.html

Reply:
TIL, thanks for the tip!